eox_tenant/tenant_wise/proxies.py

```diff
@@ -160,3 +163,62 @@ def set_key_to_cache(cls, key, value):
             value,
             EOX_TENANT_CACHE_KEY_TIMEOUT
         )
+
+
+class TenantCertificateManager(models.Manager):
+    """
+    Custom Manager Class that allows to override the get_queryset
+    method in order to filter the results by org.
+    """
+
+    def get_queryset(self):
+        """
+        Call parent method and filter the certificates by org.
+        """
+        generated_certificates = super(TenantCertificateManager, self).get_queryset()
+        excluded_courses = [
+            certificate.course_id
+            for certificate in generated_certificates
+            if certificate.course_id.org not in getattr(settings, "course_org_filter", [])
```

Comment:
I think that we should create a function that returns

getattr(settings, key)

If we do that we can differentiate the times when we want to get a value for the current tenant with the times when we want to get general django setting.

Reply:
Is there ever a case outside of the internals of the eox-tenant library where we want to get the general django settings instead of the current tenant. I think not, but I would be happy to learn about such cases

Reply:
By internals of eox-tenant I mean the part where we load the current tenant in the settings object. This proxy would not be part of the internals

Reply:
I think that right now there is not such case. But that change would do this more similar to the configuration_helpers used in openedx and any other projects could import that tenant_helpers module easily.

Reply:
@morenol  I think a goal for this plugin is to do the opposite, do the things the least similar to configuration helpers in order to avoid the issues we have with it.

Reply:
It is correct. An unspoken rule of design here was to go against the way site_configuration works